featurebyte/query_graph/sql/tile_compute.py

```diff
@@ -166,10 +167,12 @@ def update_max_window_size(self, tile_info: TileGenSql) -> None:
         tile_id = tile_info.tile_table_id
         for window in tile_info.windows:
             if window is not None:
-                window_size = int(pd.Timedelta(window).total_seconds())
+                window_size = parse_duration_string(window)
                 assert window_size % tile_info.frequency == 0
             else:
                 window_size = None
+            if window_size is not None and tile_info.offset is not None:
```

Comment:
:nit Would it be more readable if we put this condition inside the if block above?